pom.xml

```diff
@@ -15,8 +15,9 @@
         <revision>0.0.1</revision>
         <changelist>-SNAPSHOT</changelist>
         <!-- Baseline Jenkins version you use to build the plugin. Users must have this version or newer to run. -->
-        <jenkins.version>2.277.1</jenkins.version>
-        <java.level>8</java.level>
+        <jenkins.version>2.452.3</jenkins.version>
+        <java.level>11</java.level>
```

Comment:
java.level is deprecated. Should be removed

Reply:
Thanks!